ip_map_branch.txt

```diff
@@ -28,17 +28,17 @@ seven_c	https://github.com/openemr/openemr.git	rel-501	0	1	0	0	demo_5_0_0_5.sql
 seven_d	https://github.com/openemr/openemr.git	master	0	0	1	0	0	0	1	seven.openemr.io/d	hey	branch	0	0	0	0	OPEN
 seven_e	https://github.com/openemr/openemr.git	rel-502	0	0	1	0	0	0	1	seven.openemr.io/e	hey	branch	0	1	0	0	Public OpenEMR 5.0.2 Development Demo on Ubuntu 18.04
 seven_f	https://github.com/openemr/openemr.git	rel-502	0	0	0	0	demo_5_0_0_5.sql	0	1	seven.openemr.io/f	hey	branch	5.0.0	1	0	0	Public OpenEMR 5.0.2 Development Demo on Ubuntu 18.04 With Demo Data
-eight	https://github.com/openemr/openemr.git	master	0	1	0	0	0	0	1	eight.openemr.io	hey	branch	0	2	0	0	Public OpenEMR Development Demo on Alpine 3.12
-eight_a	https://github.com/openemr/openemr.git	master	0	1	0	0	demo_5_0_0_5.sql	0	1	eight.openemr.io/a	hey	branch	5.0.0	2	0	0	Public OpenEMR Development Demo on Alpine 3.12 With Demo Data
+eight	https://github.com/openemr/openemr.git	master	0	1	0	0	0	0	1	eight.openemr.io	hey	branch	0	2	0	0	OPEN
+eight_a	https://github.com/openemr/openemr.git	master	0	1	0	0	demo_5_0_0_5.sql	0	1	eight.openemr.io/a	hey	branch	5.0.0	2	0	0	OPEN
 eight_b	https://github.com/openemr/openemr.git	rel-600	0	0	0	0	0	0	1	eight.openemr.io/b	hey	branch	0	2	0	0	Public OpenEMR 6.0.0 Development Demo on Alpine 3.12
 eight_c	https://github.com/openemr/openemr.git	rel-600	0	0	0	0	demo_5_0_0_5.sql	0	1	eight.openemr.io/c	hey	branch	5.0.0	2	0	0	Public OpenEMR 6.0.0 Development Demo on Alpine 3.12 With Demo Data
 nine	https://github.com/openemr/openemr.git	master	0	1	1	0	0	0	1	nine.openemr.io	hey	branch	0	2	0	0	Public OpenEMR Development Demo on Ubuntu 20.04 (with nodejs 14)
 nine_a	https://github.com/openemr/openemr.git	master	0	1	0	0	demo_5_0_0_5.sql	0	1	nine.openemr.io/a	hey	branch	5.0.0	2	0	0	Public OpenEMR Development Demo on Ubuntu 20.04 (with nodejs 14) With Demo Data
-ten	https://github.com/openemr/openemr.git	master	0	1	0	0	0	0	1	ten.openemr.io	hey	branch	0	2	0	0	Public OpenEMR Development Demo on Alpine 3.9
-ten_a	https://github.com/openemr/openemr.git	master	0	1	0	0	demo_5_0_0_5.sql	0	1	ten.openemr.io/a	hey	branch	5.0.0	2	0	0	Public OpenEMR Development Demo on Alpine 3.9 With Demo Data
+ten	https://github.com/openemr/openemr.git	master	0	1	0	0	0	0	1	ten.openemr.io	hey	branch	0	2	0	0	Public OpenEMR Development Demo on Alpine 3.13
+ten_a	https://github.com/openemr/openemr.git	master	0	1	0	0	demo_5_0_0_5.sql	0	1	ten.openemr.io/a	hey	branch	5.0.0	2	0	0	Public OpenEMR Development Demo on Alpine 3.13 With Demo Data
```

Comment:
Agree this would be useful new demo server that would then drop:
https://www.open-emr.org/wiki/index.php/Development_Demo#Alternate_Demo_.28Alpine_3.12_-_uses_php_7.3.29

And then add new alternate demo set with Alpine 3.13 (with PHP 7.4) on the Development Demo page.